src/main/java/org/snpsift/testCases/TestCasesFilter.java

```diff
@@ -1644,4 +1644,24 @@ public void test_56_empty_vcf() {
 		Assert.assertTrue(standardOutput.contains("#CHROM\tPOS\tID\tREF\tALT"));
 	}
 
+    /**
+     * Filter: Multiple [*]
+     */
+    public void test_57() {
+        Gpr.debug("Test");
+
+        // Filter data
+        SnpSiftCmdFilter snpsiftFilter = new SnpSiftCmdFilter();
+        String expression = "(EXAC_AF[*] <= 0.1) & (COSMIC_SITE_COUNT_SOMATIC[*] >= 2)";
```

Comment:
If you add this test case and undo the other changes above, `EXAC_AF[*]` always yields 0, so the left hand side of this expression always evaluates to `true`